redfish-core/include/utils/name_utils.hpp

```diff
@@ -37,7 +37,11 @@ inline void getPrettyName(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
     // Ensure we only got one service back
     if (services.size() != 1)
     {
-        BMCWEB_LOG_ERROR("Invalid Service Size  {}", services.size());
+        BMCWEB_LOG_ERROR("Invalid Service Size {}", services.size());
+        for (const auto& service : services)
+        {
+            BMCWEB_LOG_ERROR("Invalid Service Name: {}", service.first);
```

Comment:
Would this be better as BMCWEB_LOG_DEBUG()? Seems like it could flood the log if we have a coding problem that is calling this with multiple services regularly.

Reply:
This section is a part `internalError()` [which is enabled if `asyncResp` is defined].
So, it makes sense to show it as an error.   (I think it was also the original intention )

Reply:
Agreed that makes sense.